plugins/outputs/iotdb/iotdb.go

```diff
@@ -0,0 +1,287 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package iotdb
+
+// iotdb.go
+
```

Comment:
Please remove.

Reply:
It's my bad to leave this, I will delete it.